cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -34,6 +34,11 @@ spec:
     count: 3
     allowMultiplePerNode: false
   # enable the ceph dashboard for viewing cluster status
+  mgr:
+    # list of managers module to enable
+    # available modules can be seen at https://docs.ceph.com/docs/master/mgr/
```

Comment:
We are already enabling some modules such as dashboard, rook, etc. We should point out what modules are already handled by other settings in rook.